public/locales/de/common.json

```diff
@@ -8,10 +8,16 @@
   "pages.faq": "FAQ",
   "pages.builds": "Ausrüstungen",
   "pages.analyzer": "Ausrüstungs-Analyse",
+  "pages.maps": "Arenen-Listen",
 
   "header.profile": "Profil",
   "header.logout": "Ausloggen",
   "header.login": "Einloggen",
+  
+  "auth.errors.aborted": "Einloggen abgebrochen",
+  "auth.errors.failed": "Einloggen fehlgeschlagen",
+  "auth.errors.discordPermissions": "Für dein sendou.ink-Profil wird Zugriff auf den Namen, Avatar und Social-Media-Verlinkungen in deinem Discord-Profil benötigt.",
+  "auth.errors.unknown": "Das Einloggen über Discord ist aus einem unbekannten Grund fehlgeschlagen. Falls das weiterhin passiert, kontaktiere bitte für mehr Hilfe.",
```

Comment:
These keys already exist at the bottom of the file, I think I accidentally put them into a different position than the ones in the english json 😅 Feel free to select your translations over mine where you prefer them, I think yours sound a bit more natural.

Reply:
Oh, that’s unfortunate, I didn’t see that! I also checked the translation progress MD file, but it seems it was not updated with the keys you added already.
I’ll take a look tomorrow and will pick the best of both translations!

Reply:
Huh, not sure what happened there, I could've sworn I ran the script in the end 🤔 Sorry, and thanks!

Reply:
Done - I hope it's fine now!
Someone now also has to update the MD file so hopefully it doesn't happen again 😄